frontend/src/app/components/home/home.component.ts

```diff
@@ -9,11 +10,18 @@ import { DefaultService } from 'src/assets/ts-api-client';
 export class HomeComponent implements OnInit {
   plants: any;
 
-  constructor(private defaultService: DefaultService) {}
+  constructor(
+    private defaultService: DefaultService
+  ) // private apiService: ApiService
+  {}
 
   ngOnInit(): void {
     this.defaultService.plantGet().subscribe((result) => {
+      console.log(result);
```

Comment:
Should the log call be included? Maybe check for dev environment and only log in that case, not in production.